core/metrics/src/main/scala/org/typelevel/otel4s/metrics/Measurement.scala

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2022 Typelevel
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.typelevel.otel4s.metrics
+
+import org.typelevel.otel4s.Attribute
+
+/** @param value
+  *   the value to record
+  *
+  * @param attributes
+  *   the set of attributes to associate with the value
+  */
+final case class Measurement[A](value: A, attributes: Attribute[_]*)
```

Comment:
The specific type is more favorable in a case class rather than varargs.
```suggestion
final case class Measurement[A](value: A, attributes: List[Attribute[_]])
```